src/runboat/k8s.py

```diff
@@ -282,6 +282,7 @@ def log(build_name: str, job_kind: DeploymentMode | None) -> str | None:
         corev1.read_namespaced_pod_log(
             pod.metadata.name,
             namespace=settings.build_namespace,
+            container="odoo",
```

Comment:
A slightly annoying abstraction leak, since we otherwise don't have to know the details of what is s deployed.

Wondering if there is an equivalent of kubectl log --all-containers.

Reply:
Hi @sbidoul I understand your concern about keeping runboat "deployment agnostic". As a first step, for this specific issue, I improved the fix to rely on KEP-2227 standard "kubectl.kubernetes.io/default-container" annotation (supported by kubectl but not by kubernetes python lib yet, hence the explicit `container=` param). Validated both for _deployment_ and _initialize_ logs.

Reply:
Looks like an elegant and simple solution. Thanks a lot. I'll deploy soon.